### PR TITLE
fix(ui): let's centralize the contact email address definition in a single location 📧

### LINKED ISSUE #2326: Eliminate email drama by offering a single consistent email address
## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Documentation issue or request
</code></pre>

## The problem
Some instances of the application refer to `fuse-online-tech-preview@redhat.com` as the contact email address. Some others refer to `fuse-ignite-tech-preview@redhat.com`. Users are getting confused, emails are being lost. So. Much. Drama...

## Expected behavior
Let's remove drama from our lives and make `fuse-online-tech-preview@redhat.com` our unique contact email of choice! Also, it would be super-nice to centralize the email definition in a single location so we can rapidly update it no matter where it is referenced throughout the maze of HTML stravaganza of our application.

This is a job for... SUPER REDUX I18N!!!!! 🚀

### PR DESCRIPTION
**Pull request by deeleman**
Fixes #2326

### CODE REVIEW

**Comment by TovaCohen** [comment]
I looked at the updates and they look fine to me, but someone else, who really understands the code, should approve this.

**Comment by deeleman** [comment]
Thanks Tova! Any takers? @kahboom @seanforyou23 ?

File: app/ui/src/app/app.component.html [hunk 1]
```diff
@@ -58,24 +45,21 @@
...
-                aria-labelledby="helpDropdown">
+            <ul class="dropdown-menu" id="helpDropdownMenu" aria-labelledby="helpDropdown">
+              <li>
+                <a [href]="tutorialLink" rel="nofollow" role="link" target="_blank">Sample Integration Tutorials</a>
+              </li>
+              <li>
+                <a [href]="userGuideLink" rel="nofollow" role="link" target="_blank">User Guide</a>
+              </li>
               <li>
+<<<<<<< HEAD
```
Comment (gashcrumb) [inline app/ui/src/app/app.component.html]: Are these merge conflict markers in the file?  If so these need to go...

File: app/ui/src/app/app.component.html [hunk 2]
```diff
@@ -86,12 +70,13 @@
                     rel="nofollow"
                     role="link"
                     target="_blank">User Guide</a>
+=======
```
Comment (gashcrumb) [inline app/ui/src/app/app.component.html]: This one too

File: app/ui/src/app/app.component.html [hunk 3]
```diff
@@ -86,12 +70,13 @@
...
+                <a routerLink="/support" role="link">Support</a>
+              </li>
+              <li>
+                <a href="mailto:{{ 'email' | synI18n }}" rel="nofollow" role="link" target="_blank">Contact Us</a>
+>>>>>>> fix: centralize email in a single location
```
Comment (gashcrumb) [inline app/ui/src/app/app.component.html]: And here...

**Comment by gashcrumb** [comment]
Ah, and of course thanks to me this now needs a rebase too, sorry!

**Pull request merged**